### PR TITLE
Feature/backward compatibility

### PR DESCRIPTION
**Pull request by victorshevtsov**

### CODE REVIEW

File: packages/client/src/Queries.ts [hunk 1]
```diff
@@ -145,6 +145,7 @@ function isQueryRange<T extends QueryRangeOptions>(options: any): options is T {
 export type QueryOptions = {
 	abortSignal?: AbortSignal;
 	verifyNetworkResponses?: VerificationOptions | boolean;
+	raw?: boolean;
```
Comment (rsoury) [inline packages/client/src/Queries.ts:148]: What is the significance of passing `raw` and using `PushBuffer`?

File: packages/client/src/Queries.ts [hunk 2]
```diff
@@ -357,7 +359,7 @@ export class Queries {
 		const [streamId, streamPartition] =
 			StreamPartIDUtils.getStreamIDAndPartition(streamPartId);
 		const queryString = this.httpUtil.createQueryString(query);
-		return `${baseUrl}/streams/${encodeURIComponent(
+		return `${baseUrl}/stores/${encodeURIComponent(
```
Comment (rsoury) [inline packages/client/src/Queries.ts:362]: **Note:** 
Where `/streams/` has been converted to `/stores/`

File: packages/storage-proxy/src/cli/arguments.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { toEthereumAddress } from "@streamr/utils";
+import { Argument } from 'commander';
+import { StorageNodeMetadata, } from "streamr-client";
```
Comment (rsoury) [inline packages/storage-proxy/src/cli/arguments.ts]: Are we using Prettier for opinionated format?
This comma should have been removed.

File: packages/client/src/Queries.ts (hunk 1 above)
Comment (victorshevtsov) [reply packages/client/src/Queries.ts:148]: Passing raw here, means the LogStoreClient won't decrypt the messages queried from the LogStore plugin of a broker.
Instead the LogSroreClient will forward the messages through PushBuffer to the querying StreamrClient letting it to decrypt the messages by its own. Doing so the LogStoreClient as a core thing of the StorageProxy cannot access encrypted messages, it just forwards whatever returned by the broker.

File: packages/storage-proxy/src/cli/arguments.ts (hunk 3 above)
Comment (victorshevtsov) [reply packages/storage-proxy/src/cli/arguments.ts]: Yes, what's wrong with these lines?

Comment (rsoury) [reply packages/storage-proxy/src/cli/arguments.ts]: Comma** after `StorageNodeMetadata` should be auto-formatted to be removed.
Just making sure Prettier is being used.

Comment (victorshevtsov) [reply packages/storage-proxy/src/cli/arguments.ts]: Ah, yes there was missing eslint and prettier configs.
I pushed configs and auto formatted sources for the storage-proxy package.

**Pull request merged**